### ISSUE TITLE
VFB render last button ignoring scene changes

### ISSUE THREAD

**Issue by headsmell** [ChaosGroup/vray-for-houdini#134]
The vray frame buffer "render last" teapot button ignores scene changes. When adjusting materials or moving transforms on a object, the render last button still renders previous settings. Only when the rendering from the vray_renderer node does the changes take effect. 

windows 7 / vfh-a19f141

**Comment by blented** [comment]
Seeing this here as well, seems like the .vrscene just isn't getting re-exported?

**Comment by bdancer** [comment]
We do not support actions for this button yet.

**Comment by dominik2608** [comment]
Hi, is there any progress on that issue - just curious...

**Comment by bdancer** [comment]
@dominik2608 have u tried pressing it in the latest builds? Afair I've implemented this last week, should work...

**Comment by dominik2608** [comment]
Yes, I use the build from today. For example render setting changes are not transferred and the buckets change from triangulation to top/bottom. In one of my scenes I get a thin black line and fireflies... but this would be difficult to reproduce I think atm....

**Comment by galagast** [comment]
I tested this out on a simple scene.
Using build: vfh 1809 3ab87a2 hfs16.0.736 2017-11-01
Seems to be working on my end. I'll look out for this next time I try to setup a more complex scene.
